### PR TITLE
Updates example

### PR DESCRIPTION
**Pull request by javl**
Small update to the example; adds a simple animation underneath each of
the plots showing the different easing effects.

### CODE REVIEW

File: example_plots/src/ofApp.cpp [hunk 1]
```diff
@@ -2,8 +2,14 @@
 
 //--------------------------------------------------------------
 void ofApp::setup(){
-	//ofSetBackgroundColor(255);
+	ofBackground(255);
```
Comment (arturoc) [inline example_plots/src/ofApp.cpp]: It's preferred to use ofSetBackgroundColor now

**Review by arturoc — CHANGES_REQUESTED** [review]
Can you post a screenshot to understand what it does? I've also added a comment

**Comment by javl** [comment]
I've changed `ofBackground` to `ofSetBackgroundColor`

The update adds a small animated rectangle underneath each plot, showing its easing behavior (moves in 3 seconds, waits for 1, then restarts movement).


unnamed window_340

**Comment by arturoc** [comment]
that looks great thanks!

**Pull request merged**